pages/common/ogrmerge.py.md

```diff
@@ -0,0 +1,16 @@
+# ogrmerge.py
+
+> Merge several vector datasets into a single one.
+> More information: <https://gdal.org/programs/ogrmerge.html>.
+
+- Create a GeoPackage with a layer for each input Shapefile:
+
+`ogrmerge.py -f {{GPKG}} -o {{path/to/output}}.gpkg {{path/to/input_a.shp path/to/input_b.shp ...}}`
```

Comment:
Is the `.gpkg` mandatory? otherwise should be inside the token syntax
```suggestion
`ogrmerge.py -f {{GPKG}} -o {{path/to/output.gpkg}} {{path/to/input1.shp path/to/input2.shp ...}}`
```